timemachine/fe/free_energy.py

```diff
@@ -1407,17 +1407,17 @@ def format_rate(r):
                 return f"{r * 100.0:5.1f}%"
 
             def format_rates(rs):
-                return " | ".join(format_rate(r) for r in rs)
+                return " |".join(format_rate(r) for r in rs)
```

Comment:
Minor but intentional: there's a spacing issue in the current HREX print statements where there's an extra space to the right of the "|" character, eg: ` 26.3% |  28.%` because the formatting is for 3 digits to the left of `.` - I was trying to make it more compact